packages/design-system-dashboard-cli/src/find-ds-components.js

```diff
@@ -106,6 +114,23 @@ function tallyResults(vwComponents, contentBuildWC) {
     });
   });
 
+  uswdsComponents.forEach(i => {
+    i.matches.forEach(m => {
+      const componentName = m[1];
+      if (componentName) {
+        incrementUswdsCount(componentName);
+
+        // If the component is not yet in the collection, initialize it as an empty array
+        if (!instancesByComponent[componentName])
+          instancesByComponent[componentName] = [];
+
+        // Only include the app path if it hasn't already been included previously
+        if (!instancesByComponent[componentName].includes(i.path))
+          instancesByComponent[componentName].push(i.path);
+      }
+    });
+  });
```

Comment:
Does a count of the queried uswds components. I couldn't think of an easy way to modify the first query that returns vwComponents, so the query just runs again.